README.md

```diff
@@ -56,7 +57,8 @@ NAME:
    uva test - test code locally
 
 USAGE:
-   uva test FILE
+   uva test {id}.{name}.{ext}}
+   uva test 10041.happy.cpp
```

Comment:
The filename is generated by `uva touch 10041`, so I think it's OK to assume the format is right. `{id}.{name}.{ext}` may not be clear that what `uva test` needs is a file.

Reply:
oh, because in my use case, I doesn't know about `uva touch` command, so I create file manually, maybe we can add hint about `uva touch` into README ?

Reply:
Since there's all subcommands listed in README.md, I think it's both OK to add hint or not and it's up to you. Maybe print the hint when fail to parse filename?